sdks/java/harness/src/main/java/org/apache/beam/runners/core/BeamFnDataReadRunner.java

```diff
@@ -47,6 +50,10 @@
 public class BeamFnDataReadRunner<OutputT> {
   private static final Logger LOG = LoggerFactory.getLogger(BeamFnDataReadRunner.class);
 
+  private final ObjectMapper mapper =
```

Comment:
You can use a plain ObjectMapper since we only need to support custom user types on PipelineOptions